cves/kernel/CVE-2016-4440.yml

```diff
@@ -326,8 +352,12 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    Signed-off-by: Roman Kagan <rkagan@virtuozzo.com>
```

Comment:
How were there people that signed off on the fix when you couldn't find the fix yourself?

Reply:
This is located in the git commit via the linux git tracker.  I copy and pasted those notes from the git tracker onto the CVE